native/src/main/java/io/ballerina/stdlib/graphql/runtime/schema/FieldFinder.java

```diff
@@ -269,6 +306,15 @@ private void setTypeForField(MethodType method, SchemaField schemaField) {
         }
     }
 
+    private void setTypeForField(Type resourceReturnType, SchemaField schemaField) {
```

Comment:
Why can't we use `setTypeForField(MethodType method, SchemaField schemaField)` here? We are basically doing the same thing, right?